openapi3/openapi3.go

```diff
@@ -87,6 +87,18 @@ func (doc *T) AddServer(server *Server) {
 	doc.Servers = append(doc.Servers, server)
 }
 
+func (doc *T) AddServers(servers ...*Server) {
+	doc.Servers = append(doc.Servers, servers...)
+}
+
+func (doc *T) AddServersSlice(servers []*Server) {
+	doc.Servers = append(doc.Servers, servers...)
+}
```

Comment:
```suggestion
```

Given `func (doc *T) AddServers(servers ...*Server) {` this is useless.

Reply:
ok thats fine i ll go ahead and remove it